Automattic/jetpack
# Fixes the production build and removes dev dependencies from it.

Fixes the production build command that was broken. Before you would have to install every dependency to build Jetpack, now only needed dependencies are required and installed on `yarn build-production`.

#### Changes proposed in this Pull Request:

* Fixed the production build CLI command.
* Removed development dependencies from the production build.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* This is an attempt to fix the broken Beta Builder by reducing the number of binaries we build on `yarn build`.

#### Testing instructions:






* Try to `yarn build`, you should get a working Jetpack installation.
* Try to `yarn build-production`, you should get a working Jetpack installation with minified code and less installed dependencies. Some things will be broken - like linting, commit checks, etc, but these are not necessary for production builds.

#### Proposed changelog entry for your changes:

* Fixed the CLI command to build Jetpack in production mode.
